src/static/js/templates/prepare-worksheets/input-graded.hbs

```diff
@@ -3,16 +3,16 @@
     <section class="input-with-btns">
       <div class="input-with-btns_btns">
         <label><span class="u-visually-hidden">Priority level: choose one</span>
-          <button class="btn btn__grouped-first {{#if highGrade}}active{{/if}}">{{highGradeText}}</button>
-          <button class="btn btn__grouped {{#if mediumGrade}}active{{/if}}">{{mediumGradeText}}</button>
-          <button class="btn btn__grouped-last {{#if lowGrade}}active{{/if}}">{{lowGradeText}}</button>
-          {{#if deletable}}
+          <button class="btn btn__grouped-first {{#if is_0}}active{{/if}}">{{grades.[0]}}</button>
```

Comment:
It's created as a property of the data object passed into the template, but there's really no need for the underscore. I'll remove it.